test/cypress/e2e/multi-datasources/multi_datasources_enabled.spec.js

```diff
@@ -98,7 +98,7 @@ describe('Multi-datasources enabled', () => {
     cy.get('[data-test-subj="dataSourceSelectableContextMenuHeaderLink"]').contains('9202');
   });
 
-  it('Checks Users Tab', () => {
+  it.skip('Checks Users Tab', () => {
```

Comment:
Skipping the flaky test, will re-enable when reading from url to avoid unnecessary component clicks. #1858